### PR TITLE
Update distribution.md

### PR DESCRIPTION
**Pull request by icarusdust**
Testing .aab via FAD requires to be linked to Google Play

### CODE REVIEW

File: content/yaml-publishing/distribution.md [hunk 1]
```diff
@@ -256,6 +256,8 @@ publishing:
 
 If you wish to pass release notes with your build, create a `release_notes.txt` file and add it to the project working directory, which is either the repository root directory or the Project path specified in the Build section in your workflow settings. Codemagic will fetch the content of that file and publish it with the build.
 
+In order to distribute `.aab` to testers via Firebase App Distribution, Google Play connection is required. More information [here](https://firebase.google.com/docs/app-distribution/android/distribute-console?apptype=aab#before_you_begin).
```
Comment (helinanever) [inline content/yaml-publishing/distribution.md:259]:
```suggestion
In order to distribute an `.aab` to testers via Firebase App Distribution, your Firebase project must be linked to your Google Play account. More information is available [here](https://firebase.google.com/docs/app-distribution/android/distribute-console?apptype=aab#before_you_begin).
```

Comment (icarusdust) [reply content/yaml-publishing/distribution.md]: changes made

File: content/yaml-publishing/distribution.md [hunk 2]
```diff
@@ -256,6 +256,8 @@ publishing:
 
 If you wish to pass release notes with your build, create a `release_notes.txt` file and add it to the project working directory, which is either the repository root directory or the Project path specified in the Build section in your workflow settings. Codemagic will fetch the content of that file and publish it with the build.
 
+In order to distribute an `.aab` to testers via Firebase App Distribution, your Firebase project must be linked to your Google Play account. More information is available [here](https://firebase.google.com/docs/app-distribution/android/distribute-console?
+
```
Comment (helinanever) [inline content/yaml-publishing/distribution.md:260]: the link is incomplete and the sentence is missing a full stop

**Review by helinanever — COMMENTED** [review]
the incomplete link needs to be fixed

File: content/yaml-publishing/distribution.md (hunk 2 above)
Comment (icarusdust) [reply content/yaml-publishing/distribution.md:260]: attached linked should be fine now

**Review by helinanever — APPROVED** [review]
LGTM

File: content/yaml-publishing/distribution.md [hunk 3]
```diff
@@ -176,12 +176,17 @@ publishing:
 
 If you wish to pass release notes with your build, create a `release_notes.txt` file and add it to the project working directory, which is either the repository root directory or the Project path specified in the Build section in your workflow settings. Codemagic will fetch the content of that file and publish it with the build.
 
+In order to distribute an `.aab` to testers via Firebase App Distribution, your Firebase project must be linked to your Google Play account. More information is available [here](https://firebase.google.com/docs/app-distribution/android/distribute-console?apptype=aab#before_you_begin)
+
+### Publishing an app with Fastlane
```
Comment (helinanever) [inline content/yaml-publishing/distribution.md:181]: looks like this heading is unnecessary

**Pull request merged**